packages/binding-opcua/src/opcua-protocol-client.ts

```diff
@@ -63,15 +64,15 @@ export type NodeIdLike2 = NodeIdLike & {
 };
 
 export interface FormPartialNodeDescription {
-    "opcua:nodeId": NodeIdLike | NodeByBrowsePath;
+    "opcua:nodeId"?: NodeIdLike | NodeByBrowsePath;
```

Comment:
General question/comment:
Why do we have interface FormPartialNodeDescription and OPCUAFormInvoke.
It is somewhat similar without extending (OPCUA)Form.

Moreover, OPCUAForm extends FormPartialNodeDescription again !?
Some dependencies I don't understand, and I wonder whether we can simplify it.

@erossignon

Reply:
I simplified it, and we do not see any problems when testing via unit tests nor when testing with real devices